### PR TITLE
[minor] clean extccomp

### PR DESCRIPTION
**Pull request by ringabout**

### CODE REVIEW

File: compiler/extccomp.nim [hunk 1]
```diff
@@ -12,9 +12,9 @@
 # from a lineinfos file, to provide generalized procedures to compile
 # nim files.
 
-import
-  ropes, os, strutils, osproc, platform, condsyms, options, msgs,
-  lineinfos, std / sha1, streams, pathutils, sequtils, times, strtabs
+import ropes, platform, condsyms, options, msgs, lineinfos, pathutils
+
+import std/[os, strutils, osproc, sha1, streams, sequtils, times, strtabs, json]
```
Comment (konsumlamm) [inline compiler/extccomp.nim:17]:
```suggestion
import platform, condsyms, msgs, lineinfos, pathutils

import std/[ropes, os, strutils, osproc, options, sha1, streams, sequtils, times, strtabs, json]
```
`ropes` and `options` are also standard library modules.

Comment (ringabout) [reply compiler/extccomp.nim:17]: not really, `ropes` and `options` are compiler modules too(they do in this module).

Comment (konsumlamm) [reply compiler/extccomp.nim:17]: Ah, my bad.

Comment (ringabout) [reply compiler/extccomp.nim:17]: never mind

**Pull request merged**